src/app/models/myopenstax-user.ts

```diff
@@ -20,31 +20,32 @@ type OSUser = {
         createdAt: string;
         salesforceId: string;
     };
-}
+};
 
 async function fetchUser() {
     const user = await sfApiFetch('users');
 
     return camelCaseKeys({...INITIAL_STATE, ...user}) as OSUser;
 }
 
-export default function useMyOpenStaxUser(isVerified: boolean, fetchTime: unknown) {
+export default function useMyOpenStaxUser(
+    isVerified: boolean,
+    fetchTime: number
+) {
     const [user, setUser] = React.useState<OSUser>({error: 'not loaded'});
-    const {flags: {my_openstax: isEnabled}} = useSharedDataContext() as unknown as {
-        flags: {
-            my_openstax: boolean
-        }
-    };
+    const {flags} = useSharedDataContext();
 
     React.useEffect(() => {
+        const isEnabled = flags && flags.my_openstax;
```

Comment:
`flags` can be false, rather than an object or undefined, so can't do `flags?.my_openstax`